sdks/java/container/boot.go

```diff
@@ -138,6 +138,7 @@ func main() {
 		filepath.Join(jarsDir, "beam-sdks-java-harness.jar"),
 		filepath.Join(jarsDir, "beam-sdks-java-io-kafka.jar"),
 		filepath.Join(jarsDir, "kafka-clients.jar"),
+		filepath.Join(jarsDir, "postgresql.jar"),
```

Comment:
If the python tests work already is this line even required? Isn't this supposed to be handled by he artifact fetching step? 
We may want to clarify this @chamikaramj and @ihji. Does adding this line here mean we're requiring all Java jobs to have this Jar?

Reply:
We don't need it. I figured it out later that the corresponding JAR will be pulled up that matches the driver name used in Read/Write transform.